File: Programming Basics/JavaScript/Introduction/Labs/pet-shop.js
function solve(input) {
    const dogFoodPrice = 2.5;
    const catFoodPrice = 4;

    const dogs = Number(input[0]);
    const cats = Number(input[1]);

    const totalCosts = dogs * dogFoodPrice + cats * catFoodPrice;
    console.log(`${totalCosts} lv.`);
}

File: Programming Basics/C++/Introduction/Labs/PetShop/PetShop.cpp
#include <iostream>
using namespace std;

int main()
{
	const double dogFoodPrice = 2.5;
	const double catFoodPrice = 4;

	int dogs, cats;
	cin >> dogs >> cats;

	double totalCosts = dogs * dogFoodPrice + cats * catFoodPrice;
	cout << totalCosts << " lv." << endl;
}
